internal/services/privatedns/private_dns_zone_virtual_network_link_resource.go

```diff
@@ -75,6 +76,17 @@ func resourcePrivateDnsZoneVirtualNetworkLink() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"resolution_policy": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice(virtualnetworklinks.PossibleValuesForResolutionPolicy(), false),
+				DiffSuppressFunc: func(k, old, new string, d *pluginsdk.ResourceData) bool {
+					// If the API returns the Default resolution policy and no
+					// resolution_policy set in the configuration, suppress the diff
+					return old == string(virtualnetworklinks.ResolutionPolicyDefault) && new == ""
+				},
```

Comment:
instead of suppressing the diff could we make this computed? 

if that does would please leave a `// NOTE: O+C` comment explaining why it is optional + computed

Reply:
@EppO this is still outstanding, possible to take a look?